main.py

```
@@ -1,57 +1,39 @@
+        self.url = url
+        self.HEADERS = {'Accept-Language': "en-US,en;q=0.9",
+                            'User-Agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36 Edg/107.0.1418.35"}
+
+    async def get_item_info(self) -> tuple[float, str]:
+        """     Returns:
+                [0] Price
+                [1] Title
+         """
+
+        async with aiohttp.ClientSession() as session:
+            async with session.get(self.url, headers=self.HEADERS) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)